docs/support_metrics.md

```diff
@@ -0,0 +1,30 @@
+# Support Metrics
+
+Here defines all the support metrics info of Harbor.
```

Comment:
Compatibility listed by components

Reply:
This document provides compatibility information for all Harbor components.